spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -211,14 +211,13 @@
     end
 
     context "qualifying widow/er filers" do
-      let(:intake) { create(:state_file_nj_intake, filing_status: "qualifying_widow") }
+      let(:intake) { create(:state_file_nj_intake, :df_data_qss) }
```

Comment:
[dust] - is this change needed? importing a whole profile is more test setup than just setting the `filing_status` so i'd think we'd want the less-setup route... unless I'm missing something?